auxilio/.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,29 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Can you remove files located under the .idea folder as they are specific to your IDE. Once removed, it will be good to merge!